builder.toml

```diff
@@ -88,14 +88,14 @@ description = "Ubuntu bionic base image with buildpacks for Java, .NET, NodeJS,
 [[order]]
 
   [[order.group]]
-    id = "paketo-buildpacks/nginx"
-    version = "0.0.193"
+    id = "paketo-buildpacks/java-native-image"
```

Comment:
(as with other PR on base builder) Why Java Native Image before Java? The RFC doesn't distinguish, but I'm guessing there's a right answer here?

Reply:
Confirmed that the Java Native Image should come first. See https://paketobuildpacks.slack.com/archives/C0124SD3GTG/p1604414353040200